src/api/common_api.js

```diff
@@ -1269,7 +1269,8 @@ module.exports = {
 
         fs_backend: {
             type: 'string',
-            enum: ['CEPH_FS', 'GPFS', 'NFSv4']
+            enum: ['CEPH_FS', 'GPFS', 'NFSv4', '']
```

Comment:
this is a hacky solution, please avoid using it if possible as I did in https://github.com/noobaa/noobaa-core/pull/7698

Reply:
Leaving this value as an empty string does not match the schema.
It's either changing the schema to the needs (using an empty string to describe removal of fs_backend) or setting the default POSIX and adding it to the enum, as mentioned in the error:
https://github.com/noobaa/noobaa-core/blob/d18db4308db95df162ebe10b442cf110aae0f2fd/src/manage_nsfs/manage_nsfs_cli_errors.js#L281

Reply:
fs_backend is not required, please make it undefined, there is no need to add anything to the schema